tests/test_other.py

```diff
@@ -9117,3 +9117,14 @@ def test_legacy_settings_strict_mode(self):
     proc = run_process(cmd + ['-s', 'STRICT=1'], stderr=PIPE, check=False)
     self.assertNotEqual(proc.returncode, 0)
     self.assertContained('legacy setting used in strict mode: SPLIT_MEMORY', proc.stderr)
+
+  def test_safe_heap_log(self):
+    self.set_setting('SAFE_HEAP')
+    self.set_setting('SAFE_HEAP_LOG')
+    self.set_setting('EXIT_RUNTIME')
+    src = open(path_from_root('tests', 'hello_world.c')).read()
+    self.do_run(src, 'SAFE_HEAP load: ')
```

Comment:
Maybe this test was intended to be placed in `core` and not `other`? (I don't think `set_setting` and `do_run` work outside the core tests?)

Reply:
I thought it made more sense in other, since "core" has mode already there all tests are SAFE_HEAP.   

`do_run` and `set_setting` both work fine in all test suites as far as i can tell.